### DISCUSSION TITLE
Short context of llava?

### DISCUSSION THREAD

**Discussion by arkohut** [ggml-org/llama.cpp#5656] [Q&A] [answered]
![screenshot2024-02-20 18 32 12](https://github.com/ggerganov/llama.cpp/assets/39525455/7d76a44b-592b-4e91-9daf-175f3cbbdac9)

I have a question about the context using llava 1.6. Does the screenshot means that the total context of it is `2048` token? It seems that the output even not finished before using out the context.

Here is the command:

```sh
./llava-cli -m ~/ggml-yi-34b-f16-q_5_k.gguf \
  --mmproj ~/mmproj-llava-34b-f16-q6_k.gguf \
  --image ~/00263.jpg \
  -e \
  -p '<|im_start|>system\nAnswer the questions.<|im_end|><|im_start|>user\n<image>\nProvide a full description.<|im_end|><|im_start|>assistant\n'
```

**Comment by supportend** [comment, marked as answer]
I don't know this specific model, in my output i see:
`llm_load_print_meta: n_ctx_train      = 4096`
That means, i can set it to 4096 with:
`-c 4096`

**Reply by arkohut** [reply]
You are absolutely right, thanks for replying my stupid question...I suddenly forget this critical parameter.
